src/psyclone/psyir/nodes/__init__.py

```diff
@@ -39,6 +39,7 @@
 ''' PSyIR nodes package module '''
 
 from psyclone.psyir.nodes.node import colored, Node
+from psyclone.psyir.nodes.scoping_node import ScopingNode
```

Comment:
This is imported but not listed in the `__all__` list. Is that deliberate?

Reply:
No, my mistake, added on `__all__` now.